OutboundSDK/src/main/java/io/outbound/sdk/RequestHandler.java

```diff
@@ -49,11 +49,11 @@ public RequestHandler(String name, Application app, String apiKey) {
      * @param name name for the handler
      * @param app the host application
      * @param apiKey Connect private api key
-     * @param testUrl the url for the interceptor redirect
+     * @param testClient OkHttpClient for testing
      */
-    RequestHandler(String name, Application app, String apiKey, String testUrl) {
+    RequestHandler(String name, Application app, String apiKey, OkHttpClient testClient) {
```

Comment:
Can u annotate this constructor as `@VisibleForTesting` and then invoke it in the default constructor passing in OkHttpClient.